dbfit-java/core/src/main/java/dbfit/api/DbObject.java

```diff
@@ -1,12 +1,11 @@
 package dbfit.api;
 
-import dbfit.fixture.StatementExecution;
 import dbfit.util.DbParameterAccessor;
 import dbfit.util.Direction;
 
 import java.sql.SQLException;
 
 public interface DbObject {
-    public StatementExecution buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException ;
+    public DbStatement buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException;
```

Comment:
Should we align names like `buildPreparedStatement` to `buildPreparedDbStatement`?